packages/redux/src/wishlists/selectors/wishlistsSets.ts

```diff
@@ -197,7 +202,9 @@ export const getWishlistSet = createSelector(
 export const getWishlistSets = createSelector(
   [getWishlistSetsIds, state => state],
   (wishlistSetsIds, state) =>
-    wishlistSetsIds?.map(setId => getWishlistSet(state, setId)),
+    wishlistSetsIds?.map(setId =>
+      getWishlistSet(state, setId),
+    ) as WishlistSetsHydrated,
```

Comment:
If this can return undefined I think we should add ` | undefined` here :)

Reply:
Done 🙂